packages/lisk-p2p/test/unit/validation.ts

```diff
@@ -13,10 +13,21 @@
  *
  */
 import { expect } from 'chai';
-import { validatePeerAddress, validatePeerInfo } from '../../src/validation';
+import {
+	validatePeerAddress,
+	validatePeerInfo,
+	validatePeerInfoList,
+	validateRPCRequest,
+	validateProtocolMessage,
+} from '../../src/validation';
+import { PeerInfo } from '../../src/peer';
```

Comment:
`PeerInfo` no longer exists. It has been replaced with `P2PPeerInfo` and now exists in the `p2p_types.ts` file.
This was done to make it more consistent with the related `P2PNodeInfo` type and also because the `P2PPeerInfo` type is exposed by the library through the `p2p.getNetworkStatus()` method.